mautic/mautic
# Prevented SQL error for page hits report that included Hit Device

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | n
| Related user documentation PR URL | na
| Related developer documentation PR URL | na
| Issues addressed (#s or URLs) | Not reported that I know of
| BC breaks? | n
| Deprecations? | n

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 

### Required
#### Description:

A SQL error occurs when `Hit Device` is included in a page hit report. 

#### Steps to test this PR:
1. Apply the PR
2. Create/edit a report based on page hits
3. Add `Device` to the report and save.
4. The report should be viewable.

### As applicable
#### Steps to reproduce the bug:
1. Before the PR, create/edit a page hit based report and add `Hit Device` to the selected columns list.
2. Save and should get the error.
